### PR TITLE
Disable eBPF compilation in different platforms

### PR DESCRIPTION
**Pull request by thiagoftsm**
##### Summary
This PR is bringing the same fix added by @Ferroin [here](https://github.com/netdata/netdata/commit/f6d966909bb5c443e771d312f1686ec3fa2c2aff) for Netdata source-code.

We have plans to bring eBPF plugin for different platforms, but while it is not merged and considering the reported problems per Slack when we use the latest `libbpf`, the safest is to disable.

##### Component Name
netdata-installer
##### Test Plan

1 - Compile this PR on `X86_64`, and verify that `eBPF.plugin` is compiled and it runs normally
2 - Compile this PR on different platform, and verify that `eBPF.plugin` is not compiled.

### CODE REVIEW

**Review by UmanShahzad — APPROVED** [review]

**Review by ilyam8 — APPROVED** [review]

**Comment by Ferroin** [comment]
This ideally should not have been merged without a review from a code owner. The general premise is correct, but we most certainly _do not_ want to hard fail on non-x86 platforms when users haven’t explicitly disabled eBPF.

I’ll have a follow up PR later today to fix this, but in the short term expect some users to report updater issues.

**Comment by ilyam8** [comment]
> This ideally should not have been merged without a review from a code owner.

👍 

> but we most certainly do not want to hard fail 

@thiagoftsm just updated `should_install_ebpf` function, so we just don't install the `ebpf.plugin` plugin and log "Currently eBPF is only supported on Linux on X86_64.". What do you mean by _hard fail_? 🤔

**Comment by Ferroin** [comment]
> > This ideally should not have been merged without a review from a code owner.
> 
> 
> 
> > but we most certainly do not want to hard fail
> 
> @thiagoftsm just updated `should_install_ebpf` function, so we just don't install the `ebpf.plugin` plugin and log "Currently eBPF is only supported on Linux on X86_64.". What do you mean by _hard fail_?

Ah, no, you’re correct. I mistook `run_failed` for `fatal`. This should be fine as is.

**Comment by ilyam8** [comment]
>  I mistook run_failed for fatal

Actually did the same and had to look for `run_failed` function before approving the PR 😄

**Comment by thiagoftsm** [comment]
> > This ideally should not have been merged without a review from a code owner.
> 
> +1
> 
> > but we most certainly do not want to hard fail
> 
> @thiagoftsm just updated `should_install_ebpf` function, so we just don't install the `ebpf.plugin` plugin and log "Currently eBPF is only supported on Linux on X86_64.". What do you mean by _hard fail_? thinking

The `hard fail` was not written by me, this is ancient text.

**Pull request merged**
